### PR TITLE
Made Draft and Global components

### PR DESCRIPTION
**Pull request by iturnmyscreenon**
Draft component with (DraftPage/Loader/PlayerSelection) 
Global component with (Card/MiniCard)
// Modified Div into Section

### CODE REVIEW

File: frontend/src/components/Draft/DraftPage.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      setDraftRound={setDraftRound}
+      setValidIds={setValidIds}
+    />
+  );
+}
+
+DraftPage.propTypes = {
+  draftRound: PropTypes.number.isRequired,
+  setDraftRound: PropTypes.func.isRequired,
+  cardSelected: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.shape))
```
Comment (rocambille) [inline frontend/src/components/Draft/DraftPage.jsx:67]: cardSelected is not an array of object of shape, it's an array of shape. and in shape, you have to detail the properties. something like that :

    cardSelected: PropTypes.arrayOf(PropTypes.shape({
      foo: PropTypes.string.isRequired,
      bar: PropTypes.number.isRequired,
      subObject: PropTypes.shape({
        ...
      }),
      ...
    }),

File: frontend/src/components/Draft/PlayersSelection.jsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+}) {
+  return (
+    <div className="flex flex-col h-screen">
+      <div className="text-yellow-700 h-[20%] bg-black">
+        <div className="grid grid-cols-6 h-[90%] w-full overflow-hidden">
+          {cardComputer.map((el) => (
+            <MiniCard dataDeck={el} />
+          ))}
+        </div>
+        <h2 className="text-center h-[10%]">Opponent Deck</h2>
```
Comment (rocambille) [inline frontend/src/components/Draft/PlayersSelection.jsx:25]: there is a title here

File: frontend/src/components/Draft/PlayersSelection.jsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            <MiniCard dataDeck={el} />
+          ))}
+        </div>
+      </section>
+    </div>
+  );
+}
+
+PlayersSelection.propTypes = {
+  cardSelected: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.shape))
```
Comment (rocambille) [inline frontend/src/components/Draft/PlayersSelection.jsx:71]: fix prop types definition

File: frontend/src/components/Global/Card.jsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+            speed: PropTypes.number,
+            durability: PropTypes.number,
+            power: PropTypes.number,
+            combat: PropTypes.number,
+          })
+        ),
+        image: PropTypes.shape({ url: PropTypes.string }),
+      })
+    )
+  ).isRequired,
```
Comment (rocambille) [inline frontend/src/components/Global/Card.jsx:95]: that the correct way to declare prop types :)

File: frontend/src/components/Draft/DraftPage.jsx (hunk 1 above)
Comment (rocambille) [reply frontend/src/components/Draft/DraftPage.jsx:67]: good news, what you're looking for is :

    cardSelected: PropTypes.arrayOf(Card.propTypes)

(with the needed import)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/Draft/PlayersSelection.jsx [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+          PropTypes.shape({
+            intelligence: PropTypes.string,
+            strength: PropTypes.string,
+            speed: PropTypes.string,
+            durability: PropTypes.string,
+            power: PropTypes.string,
+            combat: PropTypes.string,
+          })
+        ),
+        image: PropTypes.PropTypes.objectOf(
```
Comment (Justine-Merlin) [inline frontend/src/components/Draft/PlayersSelection.jsx:85]: Same.

File: frontend/src/components/Draft/PlayersSelection.jsx [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+          PropTypes.shape({ url: PropTypes.string })
+        ),
+      })
+    )
+  ).isRequired,
+  cardComputer: PropTypes.arrayOf(
+    PropTypes.objectOf(
+      PropTypes.shape({
+        name: PropTypes.string,
+        powerstats: PropTypes.objectOf(
```
Comment (Justine-Merlin) [inline frontend/src/components/Draft/PlayersSelection.jsx:95]: Same.

File: frontend/src/components/Draft/PlayersSelection.jsx [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+          PropTypes.shape({
+            intelligence: PropTypes.string,
+            strength: PropTypes.string,
+            speed: PropTypes.string,
+            durability: PropTypes.string,
+            power: PropTypes.string,
+            combat: PropTypes.string,
+          })
+        ),
+        image: PropTypes.PropTypes.objectOf(
```
Comment (Justine-Merlin) [inline frontend/src/components/Draft/PlayersSelection.jsx:105]: Delete all others objectOf.

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

**Pull request merged**